src/schema/v2/partner.ts

```diff
@@ -96,7 +111,9 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             limit: number
             count: boolean
             offset: number
-            sort: string
+            sort?: ArticleSort
+            page: number
```

Comment:
Does the endpoint we're querying accept `page` and `size` params? I'm not super familiar with the articles API, but my understanding from [this code](https://github.com/artsy/positron/blob/5aa2670bf7ca053e3c2dd9770f6fd257cabc6ab6/src/api/apps/articles/model/retrieve.js) was that it only accepts `limit` and `offset`.

Reply:
Actually the pagination works on cursors only, so we do not need the fields so far. I removed them. Fixed